subsys/net/lib/sntp/sntp_simple.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <errno.h>
+
+#include <net/sntp.h>
+#include <net/socketutils.h>
+
+int sntp_simple(const char *server, u32_t timeout, struct sntp_time *time)
+{
+	int res;
+	static struct addrinfo hints;
+	struct addrinfo *addr;
+	struct sntp_ctx sntp_ctx;
+
+	hints.ai_family = AF_INET;
+	hints.ai_socktype = SOCK_DGRAM;
+	hints.ai_protocol = 0;
+	res = net_getaddrinfo_addr_str(server, "123", &hints, &addr);
```

Comment:
Shouldn't there be a define for the default port? Or at least a comment saying what it is?

Reply:
123 is the standard (S)NTP port per the RFC. Added comment on that.